README.md

```diff
@@ -11,16 +11,18 @@ Please see the [contributing guide](https://exercism.org/docs/building).
 Help us keep Exercism welcoming. Please read and abide by the
 [Code of Conduct](https://exercism.org/code-of-conduct).
 
-## Adding and Updating Exercises
+## Setup
+
+Use the command `just setup` provided by the `justfile`.
 
-Please use `exercise-gen.raku` in the `bin/` directory when either adding or updating an exercise.
-The `bin/` directory contains a `README.md` with information on how to use the generator.
+## Adding and Updating Exercises
 
-When adding a new exercise, ensure that the exercise is included in the `exercises` array in `config.json`.
+Use the commands `just create-practice-exercise` and `just generate-practice-exercise` provided by the `justfile`.
```

Comment:
```suggestion

Use the commands `just create-practice-exercise` and `just generate-practice-exercise` provided by the `justfile`.
```

Adds blank line after the heading.  Style should be consistent after these are applied.